src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -326,17 +304,12 @@ protected override void Dispose(bool disposing)
                 {
                     base.Dispose(disposing);
 
-                    if (_start != null)
+                    if (disposing)
                     {
-                        _accessor.SafeMemoryMappedViewHandle.ReleasePointer();
-                        _start = null;
+                        _handle.Dispose();
```

Comment:
For example, so that the next person who comes doesn't have to ask and go figure out "hunh... if this type is finalized... is it ok that we don't dispose this handle?"   

Comments are great in the code for answering the questions that people may have when they come across the code.  It prevents the problem where someone goes "hunh... is this intentional and appropriate?  or is this an oversight and a potential bug?".   When a comment can make it clear that the developer considered and appropriately thought through this, it's massively helpful for long term maintenance (especially around complex and subtle code).